### PR TITLE
fix: cursor on keyboard tool toggle

### PR DESCRIPTION
**Pull request by muzakparov**
Related to #478

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -317,6 +317,11 @@ export class App extends React.Component<{}, AppState> {
       !event.metaKey &&
       this.state.draggingElement === null
     ) {
+      if (event.key !== "7") {
+        document.documentElement.style.cursor = "crosshair";
+      } else {
+        document.documentElement.style.cursor = "text";
+      }
       this.setState({ elementType: findShapeByKey(event.key) });
```
[the remark this answers is not in the document]
Comment (vjeux) [reply src/index.tsx]: Yes please, we shouldn't hardcode the key like this.

[the remark this answers is not in the document]
Comment (muzakparov) [reply src/index.tsx]: @vjeux thanks! does it look alright?

File: src/index.tsx [hunk 2]
```diff
@@ -298,7 +304,12 @@ export class App extends React.Component<any, AppState> {
       !event.metaKey &&
       this.state.draggingElement === null
     ) {
-      this.setState({ elementType: findShapeByKey(event.key) });
+      if (shape !== "text") {
+        document.documentElement.style.cursor = CURSOR_TYPE.CROSSHAIR;
+      } else {
+        document.documentElement.style.cursor = CURSOR_TYPE.TEXT;
+      }
```
Comment (dwelle) [inline src/index.tsx]: I'd personally swap the conditional, to make it clearer (keep the logic close to the branch the condition tests for):

```js
if (shape == "text") {
    document.documentElement.style.cursor = CURSOR_TYPE.TEXT;
} else {
    document.documentElement.style.cursor = CURSOR_TYPE.CROSSHAIR;
}
```

Comment (muzakparov) [reply src/index.tsx]: @dwelle makes sense, thanks!

**Pull request merged**